brave/utility/tor/tor_impl.cc

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) 2018 The Brave Authors. All rights reserved.
```

Comment:
shouldn't this file be tor_launcher_impl.cc to match the class name?

Reply:
yep, I forgot to change the file name when I modified the class name later.

Reply:
addressed in https://github.com/brave/muon/pull/473/commits/5158b40498d344cce6528b7bd85d6efabc5aa77b